test/e2e/virtual/initializingworkspaces/virtualworkspace_test.go

```diff
@@ -109,19 +111,22 @@ func TestInitializingWorkspacesVirtualWorkspaceAccess(t *testing.T) {
 	t.Cleanup(cancelFunc)
 
 	sourceConfig := source.DefaultConfig(t)
+	sourceConfig.Wrap(func(rt http.RoundTripper) http.RoundTripper {
+		return kcp.NewClusterRoundTripper(rt)
+	})
```

Comment:
please update `server.DefaultConfig(t)` to return a wrapped config.

Reply:
and rename `DefaultConfig` to `ClusterConfig` IMO. "Default" was what Maru chose for "cluster config", i.e. an qualified config.

Reply:
We would still need the `DefaultConfig` which is the unmodified config for discovery clients. That's why haven't modified the default/source config anywhere and kept a copy around instead.